src/content/docs/products/splashkit/SplashKit Tutorials/0-overview.mdx

```diff
@@ -1,9 +1,41 @@
 ---
-title: Overview
+title: Tutorials Overview
+description: The SplashKit tutorials team works on the creation of tutorials using the SplashKit library.
 sidebar:
   order: 0
 ---
 
 import { Card, LinkCard, CardGrid, Icon } from "@astrojs/starlight/components";
 
-Placeholder page.
+The SplashKit tutorials team works on the creation of tutorials using the SplashKit library. The team is responsible for creating tutorials that cover a wide range of topics, from Graphics, Interfaces, Networking and Raspberry Pis. The tutorials are designed to help users learn how to use SplashKit and to provide them with the knowledge they need to create their own projects. Each tutorial contains an explanation of the topic, the steps involved, as well as code snippets done in C#, C++, Python and Pasal.
```

Comment:
This reads well. It may be worth looking at trying to align this introductory paragraph with what we put together in the 2.1P document:

_The SplashKit Website features tutorials aimed at helping students learn and explore SplashKit’s functionality, focusing on developing smaller, high-quality tutorials across C++, C#, and Python, rather than extensive series._

Seeing as this is a landing page, probably primarily for those onboarding, it could be worth adjusting slightly to say what we do and what our end state is (e.g. a tutorial that covers every SplashKit function).

Reply:
I've just updated this, will be in the commit I push